Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
-gem 'danger', github: 'danger/danger', :branch => 'master'
-gem 'danger-swiftlint'
+# gem 'danger', github: 'danger/danger', :branch => 'master'
```

Comment:
I would argue in favour of fully removing it instead of commenting if it is not used, wdyt?

Also, just for context, was this already not in use or removed now?

Reply:
What's the benefit of commenting/removing `danger`? 

I would suggest to leave a comment why it's been removed, so we'll be able to recap during an upgrade. As far as I see it's been involved in some of Mozilla's automation processes like linting and PR processing.

Reply:
Danger is not used in our project at the moment. It's implemented for lining and PR checks (a sort of checkboxes) in Mozilla. The Gemfile is learner without it, but happy to revert the change if you both 👍

Reply:
I am fine to have a lean Gemfile and danger removed, but as stated before please leave a comment in the Gemfile 🙏